### PR TITLE
return data to kernel

### PR DESCRIPTION
**Pull request by whilefoo**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -60,13 +60,26 @@ async function run() {
...
 
   return "No permit generated";
 }
 
+async function returnDataToKernel(repoToken: string, stateId: string, output: object) {
+  const octokit = new Octokit({ auth: repoToken });
+  await octokit.repos.createDispatchEvent({
+    owner: github.context.repo.owner,
+    repo: github.context.repo.repo,
+    event_type: "return_data_to_ubiquibot_kernel",
```
Comment (pavlovcik) [inline src/index.ts:75]: I feel that the name can be more formal. I would need to see the other event types and we can follow some type of schema/pattern.

Comment (whilefoo) [reply src/index.ts:75]: What do you suggest?

Comment (0x4007) [reply src/index.ts:75]: Where can we see a list of the other event types from GitHub?

Comment (whilefoo) [reply src/index.ts:75]: this is a custom event type, I don't think there are any examples from GitHub

**Pull request merged**